packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -109,473 +102,592 @@ const PersonalDetailsForm = ({
+                                    (values?.account_opening_reason && has_real_account)
+                                }
+                            />
+                        )}
+                    </fieldset>
+                </FormBodySection>
+            </div>
+
+            {is_svg && (
+                <div className='account-form__poi-additional-information'>
+                    <FormSubHeader title={localize('Additional information')} />
+                    {'phone' in values && (
+                        <PhoneField
+                            value={values.phone}
+                            editable_fields={editable_fields}
+                            has_real_account={has_real_account}
+                            required
+                        />
                     )}
-                </fieldset>
-            </FormBodySection>
-        </div>
+                    <React.Fragment>
+                        {'place_of_birth' in values && (
+                            <PlaceOfBirthField
+                                handleChange={handleChange}
+                                setFieldValue={setFieldValue}
+                                disabled={isFieldImmutable('place_of_birth', editable_fields)}
+                                residence_list={residence_list}
+                                required
+                            />
+                        )}
+                        {'tax_residence' in values && (
+                            <TaxResidenceField
+                                setFieldValue={setFieldValue}
+                                disabled={
+                                    isFieldImmutable('tax_residence', editable_fields) ||
+                                    (values?.tax_residence && has_real_account)
+                                }
+                                residence_list={residence_list}
+                                required
+                                setIsTaxResidencePopoverOpen={setIsTaxResidencePopoverOpen}
+                                setIsTinPopoverOpen={setIsTinPopoverOpen}
+                                is_tax_residence_popover_open={is_tax_residence_popover_open}
+                            />
+                        )}
+                        {'tax_identification_number' in values && (
+                            <TaxIdentificationNumberField
+                                is_tin_popover_open={is_tin_popover_open}
+                                setIsTinPopoverOpen={setIsTinPopoverOpen}
+                                setIsTaxResidencePopoverOpen={setIsTaxResidencePopoverOpen}
+                                disabled={
+                                    isFieldImmutable('tax_identification_number', editable_fields) ||
+                                    (values?.tax_identification_number && has_real_account)
+                                }
+                                required
```

Comment:
clarification: is tax_identification_number mandatory for all clients?
shouldnt we consider selected tax residence for this?

Reply:
As discussed, this is mandatory for all, based on tax residence will be handled from backend regardless of the TIN format & it's in discussion, for now it'll be mandatory and we've a separated card for this change in MT5 flow as well